libtock/proximity.c

```diff
@@ -0,0 +1,97 @@
+#include "proximity.h"
+#include "tock.h"
+#include <stdio.h>
+
+struct thresholds {
+  uint8_t lowerThresh;
+  uint8_t higherThresh;
```

Comment:
```suggestion
  uint8_t lower_threshold;
  uint8_t higher_threshold;
```

We generally prefer spelling out whole names; easier for future readers.

Reply:
I have made the change.